appveyor.yml

```diff
@@ -14,7 +14,7 @@ cache:
 environment:
   global:
     ELECTRON_NO_ATTACH_CONSOLE: true
-    nodejs_version: 6.1.0
+    nodejs_version: "6.10.3"
```

Comment:
I still think that we ought to document this 'required' nodejs version somewhere, even if it's only the Etcher developers who actually need to lock themselves to this version. (because we want the updated `npm-shrinkwrap.json` files to stay compatible with all our lovely CI checks)